docs/SPIRVRepresentationInLLVM.rst

```diff
@@ -74,6 +75,21 @@ mangled as __spirv_{TypeName}, where {TypeName} is the name of the SPIR-V
 type with "OpType" removed, e.g., OpTypeEvent is mapped to spirv.Event and
 mangled as __spirv_Event.
 
+Address spaces
+--------------
+
+Following
+`SPIR-V storage classes <https://www.khronos.org/registry/spir-v/specs/unified1/SPIRV.html#Storage_Class>`_
+are naturally represented as LLVM IR address spaces with following mapping:
+
+SPIR-V storage class    LLVM IR address space
```

Comment:
What about storage classes added via extensions? For example Device/Host storages from SPV_INTEL_usm_storage_classes, which are mapped on addrspace(5) and addrspace(6) appropriately?

Reply:
By 'mapped' I mean they are mapped on global_device and global_host address spaced in OpenCL (llvm.org) and in OpenCL/DPCPP (intel/llvm).

Reply:
I'm not sure I understand your question.
Do you suggest adding "the mapping" to this document or just want to learn something about these address spaces?
Could you specify the question, please?

Reply:
If I understand the document's purpose correctly, it is about 'what LLVM module can (and probably must) be correctly translated to SPIR-V. Additional requirements section introduces a restriction on what LLVM address spaces are valid. I'm saying, that for some (read: USM) extensions it's valid to have other address spaces other than 0-4 in spir LLVM IR.

Reply:
It's not clear what exactly do you imply with this comment, but I added an example with `SPV_INTEL_usm_storage_classes` extension to the document. Let me know if you expected something else.